x-pack/plugins/security_solution/public/common/components/event_details/columns.tsx

```diff
@@ -110,9 +106,7 @@ export const getColumns = ({
                 initialWidth: DEFAULT_COLUMN_MIN_WIDTH,
               })
             }
-            disabled={
-              (data.isObjectArray && data.type !== 'geo_point') || fieldFromBrowserField == null
-            }
+            disabled={data.isObjectArray && data.type !== 'geo_point'}
```

Comment:
A subtle side-effect of this change is it re-enables the checkbox for some fields, like `signal.rule.severity_mapping` in the screenshot below:

<img width="1259" alt="signal rule severity_mapping" src="https://user-images.githubusercontent.com/4459398/116945399-c054ec80-ac34-11eb-8d4e-ab4864984b02.png">